src/phantom.h

```diff
@@ -96,6 +97,8 @@ class Phantom : public QObject
 
     bool webdriverMode() const;
 
+    int remoteDebugPort() const;
```

Comment:
How can you otherwise access ```remoteDebugPort``` inside your JavaScript code ? This is used to expose ```remoteDebugPort``` to the JavaScript. Also it does use ```m_config``` to get port number: